msal/src/main/java/com/microsoft/identity/client/AuthenticationResultAdapter.java

```diff
@@ -29,9 +29,7 @@
     static IAuthenticationResult adapt(ILocalAuthenticationResult localAuthenticationResult) {
 
         IAuthenticationResult authenticationResult = new AuthenticationResult(
-                localAuthenticationResult.getAccessTokenRecord(),
-                localAuthenticationResult.getIdToken(),
-                localAuthenticationResult.getAccountRecord()
+                localAuthenticationResult.getCacheRecordWithTenantProfileData()
```

Comment:
NPE thrown here when I tried to make an ATS call with broker.

06-04 17:55:07.269 25259-25259/com.microsoft.identity.client.sample.local E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.microsoft.identity.client.sample.local, PID: 25259
    java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object java.util.List.get(int)' on a null object reference
        at com.microsoft.identity.client.AuthenticationResult.<init>(AuthenticationResult.java:48)
        at com.microsoft.identity.client.AuthenticationResultAdapter.adapt(AuthenticationResultAdapter.java:32)
        at com.microsoft.identity.client.PublicClientApplication$5.onSuccess(PublicClientApplication.java:973)
        at com.microsoft.identity.common.internal.controllers.ApiDispatcher$4$2.run(ApiDispatcher.java:418)
        at android.os.Handler.handleCallback(Handler.java:739)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)